File: tests/Generator/resource/java/import/Import.java

import com.fasterxml.jackson.annotation.JsonGetter;
import com.fasterxml.jackson.annotation.JsonSetter;

public class Import {
    private StudentMap students;
    private Student student;

    @JsonSetter("students")
    public void setStudents(StudentMap students) {
        this.students = students;
    }

    @JsonGetter("students")
    public StudentMap getStudents() {
        return this.students;
    }

    @JsonSetter("student")
    public void setStudent(Student student) {
        this.student = student;
    }

    @JsonGetter("student")
    public Student getStudent() {
        return this.student;
    }
}

File: tests/Generator/resource/rust/import/import.rs
use serde::{Serialize, Deserialize};
use student_map::StudentMap;
use student::Student;

#[derive(Serialize, Deserialize)]
pub struct Import {
    #[serde(rename = "students")]
    students: Option<StudentMap>,

    #[serde(rename = "student")]
    student: Option<Student>,

}

